invenio_records_marc21/ui/theme/assets/semantic-ui/translations/invenio_records_marc21/translations.pot

```diff
@@ -49,4 +49,7 @@ msgid "New upload"
 msgstr "New upload"
 
 msgid "Uploaded on"
-msgstr "Uploaded on"
\ No newline at end of file
+msgstr "Uploaded on"
+
+msgid "Make your first upload!"
+msgstr "Make your first upload!"
```

Comment:
minor: can we get rid of the missing newlines?

Reply:
I'll have to check